ibis/backends/snowflake/registry.py

```diff
@@ -171,9 +194,26 @@ def _unnest(t, op):
     {
         ops.JSONGetItem: fixed_arity(sa.func.get, 2),
         ops.StringFind: _string_find,
-        ops.Map: fixed_arity(sa.func.ibis_udfs.public.object_from_arrays, 2),
-        ops.MapKeys: unary(sa.func.object_keys),
-        ops.MapValues: unary(sa.func.ibis_udfs.public.object_values),
+        ops.Map: fixed_arity(
```

Comment:
Unrelated, but should we consider to use singledispatch for sqlalchemy translation functions?

Reply:
Done.

Reply:
Yeah, we could probably make that work!